script-test/playbackstrategies/modifiers/html5tests.js

```diff
@@ -0,0 +1,3260 @@
+              player.initialiseMedia(MediaPlayerBase.TYPE.VIDEO, 'http://url/', 'video/mp4', sourceContainer, {});
+
+              expect(mockVideoMediaElement.style.position).toEqual('absolute');
+              expect(mockVideoMediaElement.style.top).toEqual('0px');
+              expect(mockVideoMediaElement.style.left).toEqual('0px');
+              expect(mockVideoMediaElement.style.width).toEqual('100%');
+              expect(mockVideoMediaElement.style.height).toEqual('100%');
+              expect(mockVideoMediaElement.style.zIndex).toEqual('');
+            });
+
+            it(' Autoplay Is Turned Off On Media Element Creation', function () {
+              player.initialiseMedia(MediaPlayerBase.TYPE.VIDEO, 'http://url/', 'video/mp4', sourceContainer, {});
+
+              expect(mockVideoMediaElement.autoplay).toEqual(false);
+            });
+
+            it(' Error Event From Media Element Causes Error Log With Code And Error Message In Event', function () {
+              var errorMessage = 'This is a test error';
+
+              spyOnProperty(mockVideoMediaElement, 'error').and.returnValue({code: errorMessage});
+
+              player.initialiseMedia(MediaPlayerBase.TYPE.VIDEO, 'http://url/', 'video/mp4', sourceContainer, {});
+
+              errorCallback();
+
+              expect(logger.error).toHaveBeenCalledWith('Media element error code: ' + errorMessage);
+            });
+
+            it(' Error Event From Source Element Causes Error Log And Error Message In Event', function () {
+              var sourceError;
+              spyOn(mockSourceElement, 'addEventListener').and.callFake(function (name, methodCall) {
+                if (name === 'error') {
+                  sourceError = methodCall;
+                }
+              });
+
+              player.initialiseMedia(MediaPlayerBase.TYPE.VIDEO, 'http://url/', 'video/mp4', sourceContainer, {});
+
+              sourceError();
+
+              expect(logger.error).toHaveBeenCalledWith('Media source element error');
+            });
+
+            it(' Pause Passed Through To Media Element When In Playing State', function () {
+              player.initialiseMedia(MediaPlayerBase.TYPE.VIDEO, 'http://url/', 'video/mp4', sourceContainer, {});
+
+              metaDataCallback({});
+
+              player.beginPlayback();
+              player.pause();
+
+              expect(mockVideoMediaElement.pause).toHaveBeenCalledTimes(1);
+            });
+
+            it(' Pause Not Passed From Media Element To Media Player On User Pause From Buffering', function () {
+              player.initialiseMedia(MediaPlayerBase.TYPE.VIDEO, 'http://url/', 'video/mp4', sourceContainer, {});
+
+            // We dont fire the metadata ready callback so it stays in the buffering state
+              player.beginPlayback();
+
+              mockVideoMediaElement.pause();
+
+              expect(player.toPaused).toHaveBeenCalledTimes(0);
```

Comment:
`toPaused` should not be part of the public API, and testing that the implementation calls it is a bit white box-y. I'd suggest you assert of the effects of `toPaused` instead (event sent, state changed).

Same goes for other usages below. Also means you can stop spying on it in the `beforeEach`.